src/SfxWeb/src/app/Models/eventstore/timelineGenerators.ts

```diff
@@ -186,17 +186,37 @@ export abstract class TimeLineGeneratorBase<T> {
          throw new Error('NotImplementedError');
     }
 
-    generateTimeLineData(events: T[], startOfRange?: Date, endOfRange?: Date): ITimelineData {
+    generateTimeLineData(events: T[], startOfRange?: Date, endOfRange?: Date, nestedGroupLabel?: string): ITimelineData {
         const data = this.consume(events, startOfRange, endOfRange);
         EventStoreUtils.addSubGroups(data.groups);
+        if (nestedGroupLabel){
```

Comment:
add a comment to explain this structure really briefly and what the point of the nested group label is for.